plugins/tile/tile-plugin.cpp

```diff
@@ -271,28 +289,58 @@ class tile_output_plugin_t : public wf::pointer_interaction_t, public wf::custom
         });
     };
 
+    wf::key_callback on_toggle_maximized = [=] (auto)
+    {
+        auto view = wf::get_core().seat->get_active_view();
+        auto toplevel = toplevel_cast(view);
+        // skip if the workspace has no views
+        if (!toplevel)
+        {
+            return false;
+        }
+        auto node = tile::view_node_t::get_node(toplevel);
```

Comment:
It is possible that `node` is null here, because we allow both floating and tiled views. So if the focused toplevel happens to not be tiled, node will be null and this will crash.